hyprland-share-picker/main.cpp

```diff
@@ -84,31 +85,34 @@ int main(int argc, char* argv[]) {
     MainPicker w;
     mainPickerPtr = &w;
 
+    QSettings* settings = new QSettings("/tmp/hypr/hyprland-share-picker.conf", QSettings::IniFormat);
+    w.setGeometry(0, 0, settings->value("width").toInt(), settings->value("height").toInt());
```

Comment:
what if this is the first launch? What will be the default? Is size 0,0 ignored?

Generally I am not a fan of how huge the picker is in this MR. I'd like for the default size to remain the same. Especially with the fact it now remembers the size, if anyone wants a different one, they can resize it.

Reply:
I reverted the size to the old one already so it's not a a problem

0,0 is completely ignored

If it's the first launch it uses the default size which is the one you want

I already tested your points before doing the last commit

The default is the minimum size defined in the .ui file

When the file don't exist QSettings returns 0 when you do toInt() it's documented but because the .ui defines a minimum size the dialog is never smaller than that

Reply:
right, makes sense.

Alright. All good.